networking/synthetic/stac-s2i-builder-image/Dockerfile

```diff
@@ -7,10 +7,10 @@ ENV container docker
 ARG github_url_stac_config
 
 RUN git clone ${github_url_stac_config}
-RUN wget http://mirror.centos.org/centos/7/updates/x86_64/Packages/kernel-`uname -r`.rpm \
-    && wget http://mirror.centos.org/centos/7/updates/x86_64/Packages/kernel-devel-`uname -r`.rpm \
-    && yum install -y kernel*.rpm
-RUN yum install --assumeyes \
+RUN echo -e "[rhel-7-latesn]\nname=rhel7-latest\nbaseurl=http://download.devel.redhat.com/rel-eng/latest-RHEL-7/compose/Server/x86_64/os/\nenabled=1\ngpgcheck=0" > /etc/yum.repos.d/rhel7.repo
```

Comment:
It is provided by https://github.com/redhat-performance/perf-dept/blob/master/repo_files/rhel7-latest.repo

BTW, what do you think about make it configurable ?